src/org/labkey/test/tests/assay/UploadLargeExcelAssayTest.java

```diff
@@ -79,31 +94,71 @@ public void testUpload200kRows() throws Exception
     {
         // generate a large .tsv file
         String fileName = "200kXlsxFile.xlsx";
-        var dgen = new TestDataGenerator("samples", "chaos_sample", getProjectName());
-        dgen.withColumns(ASSAY_FIELDS)
-                .generateRows(200000);
-        var largeTsvFile = dgen.writeData("largeTsvImportFile");
-        // convert to .xlsx
-        log("converting large tsv file to .xlsx");
-        var largeXlsXFile = TestFileUtils.convertTabularToXlsx(largeTsvFile, "\t", "import", fileName);
-        log("finished converting file to xlsx");
-
+        var dgen = new TestDataGenerator("samples", "chaos_sample", getProjectName())
+            .withColumns(ASSAY_FIELDS);
+        log("writing large tsv file");
+        var largeTsvFile = dgen.writeGeneratedDataToFile(200000, "largeTsvFile.tsv");
+        log("finished writing large tsv file");
+
+        // import to new sampleType
+//        var sampleTypeDef = new SampleTypeDefinition("chaos_samples");
+//        sampleTypeDef.setNameExpression("S-${dailySampleCount}");
+//        for (PropertyDescriptor def : ASSAY_FIELDS)
+//            sampleTypeDef.addField((FieldDefinition) def);
+//        log("importing tsv to samples");
+//        SampleTypeAPIHelper.createEmptySampleType(getProjectName(), sampleTypeDef);
+//        SampleTypeHelper.beginAtSampleTypesList(this, getProjectName())
+//                .goToSampleType("chaos_samples");
+//
+//        DataRegionTable drt = DataRegionTable.DataRegion(getDriver()).waitFor();
+//        TestLogger.log("Adding data from large tsv file");
+//        ImportDataPage importDataPage = drt.clickImportBulkData();
+//        importDataPage.setFile(largeTsvFile);
+//        importDataPage.setFileInsertOption(true);
+//        clickAndWait(Locator.button("Submit"), 10 * WAIT_FOR_PAGE);
+//        log("import to samples complete");
```

Comment:
Was this supposed to be temporarily commented

Reply:
🤦 Oh, no this reflects that I tried importing the data to a sampleType briefly, then changed course to import to a separate assay.  Total goof on my part to leave that in, thanks for bringing it up